### PR TITLE
Fix load_pages management command under Python 2.

### PR DESCRIPTION
**Pull request by hodgestar**
Intended to address #373.

### CODE REVIEW

**Comment by hodgestar** [comment]
Ready for review.

File: wafer/pages/management/tests/test_load_pages.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+    def test_load_pages(self):
+        out = StringIO()
+        with self.settings(PAGE_DIR=self._page_dir + "/"):
+            call_command('load_pages', stdout=out)
+        logs = out.getvalue().splitlines()
+        self.assertEqual(sorted(logs), [
+            "Loaded page page1",
+            "Loaded page page2",
+        ])
```
Comment (drnlm) [inline wafer/pages/management/tests/test_load_pages.py]: Should we explictly test that we've actually added the pages to the database?

Comment (hodgestar) [reply wafer/pages/management/tests/test_load_pages.py]: Good idea. Check added.

**Comment by drnlm** [comment]
:+1: :

**Pull request merged**